docs/components/modeler/web-modeler/advanced-modeling/manage-connector-templates.md

```diff
@@ -0,0 +1,109 @@
+
+   ![Publishing a template](img/connector-templates/create-connector-template-3.png)
+
+5. After publishing, the template can be applied to elements of all models in the same project.
+
+   :::info Project scope
+   A Connector template is always associated with a project, hence only available in the project where you added it.
+   :::
+
+### JSON editor features
+
+The JSON editor is based on the [Monaco Editor](https://microsoft.github.io/monaco-editor/). The Monaco Editor is the editor that powers VS Code. As a result, the template editor supports many familiar features, such as auto-formatting, indentation support, code completion, and error highlighting.
+
+Read the [VS Code editor docs](https://code.visualstudio.com/docs/editor/editingevolved) for a full overview of features.
+
+### Versioning Connector templates
+
+You can version your Connector templates using [milestones](/components/modeler/web-modeler/milestones.md), similar to diagrams.
+
+If you publish a new version of a Connector template and the old version is already used in diagrams, the user can either update these diagram Connector templates to the most recent version, or maintain the older version when editing the respective diagram element (see updating [element templates](/components/modeler/desktop-modeler/element-templates/using-templates.md#updating-templates)).
+
+## Fixing problems in your templates {#fixing-template-problems}
+
+While working on a template, the template will be in invalid intermediate states. For instance, when you add a new property, it must contain various mandatory attributes.
+
+Unless all mandatory attributes are defined, the template will not be saved, and the preview is not updated. This ensures that you can never publish an invalid or broken template.
+
+The editor toolbar indicates if the template is currently in a valid state or not. The JSON editor provides you with error highlighting, allowing you to add mandatory values and resolve problems without missing anything.
+
+![Indicating problems in connector templates](img/connector-templates/fix-connector-template-problems.png)
+
+If there are problems at the root level of the JSON (such as a missing or misspelled mandatory attribute,) the error is highlighted in the first line of the editor. Click the error marker at the curly bracket to expand the error message.
+
+![Some connector template problems highlighted in the first line](img/connector-templates/fix-connector-template-problems-2.png)
+
+<!--
+## Reusing existing runtimes
+
+Not only can you create new Connector templates to bind to a job worker topic or a custom Connector SDK function, but you can also reuse [existing connector runtimes](/components/integration-framework/connectors/out-of-the-box-connectors/available-connectors-overview.md). For instance, if you add the following property and bind it to `zeebe:taskDefinition:type`, you can reuse the HTTP JSON runtime and write a prefilled Connector template to call an API easily.
```

Comment:
Minor tweak:
Update text from "Not only can you create new Connector templates to bind to a job worker topic or a custom Connector SDK function,"
to "Not only can you create new Connector templates to bind to a job worker topic or a custom connector function you've created using the Connector SDK,"